pkg/subnet/local.go

```diff
@@ -697,11 +724,33 @@ func getExpectedDefaultSnapshotSHA256Sum(isSingleNode bool) (string, error) {
 
 // Initialize default snapshot with bootstrap snapshot archive
 // If force flag is set to true, overwrite the default snapshot if it exists
-func SetDefaultSnapshot(snapshotsDir string, forceReset bool, isSingleNode bool) (bool, error) {
-	bootstrapSnapshotArchivePath := filepath.Join(snapshotsDir, constants.BootstrapSnapshotArchiveName)
-	if isSingleNode {
-		bootstrapSnapshotArchivePath = filepath.Join(snapshotsDir, constants.BootstrapSnapshotSingleNodeArchiveName)
+func SetDefaultSnapshot(snapshotsDir string, forceReset bool, avagoVersion string, isSingleNode bool) (bool, error) {
+	resetCurrentSnapshot := forceReset
```

Comment:
nit: just convert  `SetDefaultSnapshot(snapshotsDir string, resetCurrentSnapshot bool, avagoVersion string, isSingleNode bool) (bool, error) {` to `SetDefaultSnapshot(snapshotsDir string, forceReset bool, avagoVersion string, isSingleNode bool) (bool, error) {`